### PR TITLE
meta-viewport: remove delayed refresh from example

### PR DESCRIPTION
**Pull request by WilcoFiers**
There is definitely no reason why this inapplicable example should have a delayed redirect in it.

Need for Final Call: 1 week

---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by tbostic32 — APPROVED** [review]

**Review by susanatx — APPROVED** [review]

**Review by kengdoj — APPROVED** [review]

**Review by Jym77 — APPROVED** [review]

**Review by carlosapaduarte — APPROVED** [review]

**Review by colabottles — APPROVED** [review]

**Comment by patrickhlauke** [comment]
could even just replace that redirect meta with something like `<meta charset="UTF-8">` if the idea is to show an inapplicable case where there is a `meta`, but not a `viewport` one. refresh starts to conjure up unrelated thoughts of "but does this fail some other SC like timing or pause/stop/hide or similar?")

**Review by kimblim — APPROVED** [review]

File: _rules/meta-viewport-b4f0c3.md [hunk 1]
```diff
@@ -232,8 +232,8 @@ There is no viewport `meta` element.
 ```html
 <html>
 	<head>
-		<title>Simple page showing random text</title>
-		<meta http-equiv="refresh" content="10; URL='https://github.com'" />
+		<title>Redirect to W3C</title>
+		<meta http-equiv="refresh" content="0; URL='https://w3.org/'" />
```
Comment (WilcoFiers) [inline _rules/meta-viewport-b4f0c3.md:236]:
```suggestion
		<meta charset="UTF-8">
```

File: _rules/meta-viewport-b4f0c3.md [hunk 2]
```diff
@@ -232,8 +232,8 @@ There is no viewport `meta` element.
 ```html
 <html>
 	<head>
-		<title>Simple page showing random text</title>
-		<meta http-equiv="refresh" content="10; URL='https://github.com'" />
+		<title>Redirect to W3C</title>
```
Comment (WilcoFiers) [inline _rules/meta-viewport-b4f0c3.md:235]:
```suggestion
		<title>Lorem ipsum</title>
```

**Review by colabottles — COMMENTED** [review]
I've not seen this "in the wild" or implemented as such for a very, very long time. Is it still even being used? I imagine it is on legacy sites or sites that are not maintained. I think a `charset="UTF-8"` is sufficient.

**Pull request merged**